tools/tests/package-tests.js

```diff
@@ -1130,8 +1130,20 @@ selftest.define("show local package w/o version",  function () {
       version: "local",
       directory: packageDir
     });
+
+    // Test that running without any arguments also shows this package.
```

Comment:
Can we test explicitly that when changing some metadata thing locally for a published package, `meteor show` shows the local value?